cluster/saltbase/salt/kube-addons/kube-addons.sh

```diff
@@ -21,6 +21,33 @@ KUBECTL=${KUBECTL_BIN:-/usr/local/bin/kubectl}
 
 ADDON_CHECK_INTERVAL_SEC=${TEST_ADDON_CHECK_INTERVAL_SEC:-600}
 
+SYSTEM_NAMESPACE=kube-system
+
+function delete-if-present() {
```

Comment:
I'm assuming so, too. If you'd like to leave @marekbiskup a pointer to a branch somewhere with a meaningful set of code, we can use the magic of version control to preserve it, but there's no reason to keep the dead code in this PR.